index.html

```diff
@@ -21,7 +21,7 @@
 
   <div class="content text-center flow-4 relative bg-white-gradient-center">
     <div class="relative z-1">
-      {% include components/content-feature.html
+      {% include components/content-cta.html
```

Comment:
Changed the name to better reflect what they are, since the NEW component is better suited to this name. Naming is hard 😩